### ISSUE TITLE
Drop support for old Python versions

### ISSUE THREAD

**Issue by jugmac00** [morepath/more.jinja2#12]
**suggestion**
- drop support for Python 2.7
- drop support for Python 3.3 and 3.4
- add support for 3.6, 3.7, 3.8

Tests for Python 2.7 currently fail because `dectate` is using Python 3 only syntax.

Also, there is another test error:
```
___________________________ ERROR collecting more/jinja2/tests/test_jinja2.py ___________________________
import file mismatch:
imported module 'more.jinja2.tests.test_jinja2' has this __file__ attribute:
  /home/jugmac00/Projects/more.jinja2/.tox/py36/lib/python3.6/site-packages/more/jinja2/tests/test_jinja2.py
which is not the same as the test file we want to collect:
  /home/jugmac00/Projects/more.jinja2/more/jinja2/tests/test_jinja2.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
```

**Comment by hugovk** [comment]
Python 3.5 is also EOL next weekend, drop that too?

**Comment by jugmac00** [comment]
How time flies!

Dropping 3.5 is a very good idea... which also brings some possible `pyugrade` massage with it.

Do you have any thoughts about the above shown error message about test collection?

Is this about the src/not src topic? 
https://hynek.me/articles/testing-packaging/
https://blog.ganssle.io/articles/2019/08/test-as-installed.html

While the combination of namespace packages without a src top level dir is new to me, I wonder why this is the only package with this kind of problems - the other morepath packages just work (well except for linting and other errors).

**Comment by hugovk** [comment]
Is https://github.com/pytest-dev/pytest/issues/2042 the root issue? I've not read through the whole thing, but the suggestion is to use an env var to work around it:

> `py-1.7.0` has been released which supports using `PY_IGNORE_IMPORTMISMATCH=1` to suppress this error. While this is not ideal, it is at least a escape hatch for cases where it is getting in the way.

https://github.com/pytest-dev/pytest/issues/2042#issuecomment-429289164

**Comment by jugmac00** [comment]
Thanks for pointing me to that issue! That fixed the problem.

fixed by #13
